chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -318,7 +310,7 @@ object InstanceActions {
                                 "user_ids" to userNames,
                                 "private" to isPrivate
                         ).apply {
-                            if (customData != null) this += "custom_data" to customData
+//                            if (customData != null) this += "custom_data" to customData
```

Comment:
this should be uncommented and the param brought back

Reply:
👍 done.